icommons_ext_tools/requirements/base.txt

```diff
@@ -1,4 +1,4 @@
-Django==1.10.5
+Django==1.11.9
 cx-Oracle==5.2.1
```

Comment:
We should upgrade cx-Oracle too (to 6.0.3?) and possible a macro upgrade on the redis libraries. 
django-debug-toolbar in local.txt is another one.